.github/workflows/copilot_deploy.yml

```diff
@@ -20,8 +20,14 @@ on:
         type: boolean
         description: Run e2e tests
   push:
-    paths-ignore:
-      - '**/README.md'
+    paths:
+      - '!**/README.md'
+      - '**/*.py'
```

Comment:
ANY python file? In any directory? Leaving aside you could have just left it at *.py, this seems incredibly overpermissive. There's so many python files where a change wouldn't result in a complete rebuild of the service

Reply:
The python files are in the top-level on this one, rather than a subdirectory, but I've removed that - as you suggest leaving it just as *.py would be better.